packages/kit/src/exports/node/polyfills.js

```diff
@@ -32,11 +32,17 @@ const globals = {
  */
 export function installPolyfills() {
 	for (const name in globals) {
-		Object.defineProperty(globalThis, name, {
-			enumerable: true,
-			configurable: true,
-			writable: true,
-			value: globals[name]
-		});
+		const desc = Object.getOwnPropertyDescriptor(globalThis, name);
+		if (desc === undefined || desc.configurable) {
+			Object.defineProperty(globalThis, name, {
+				enumerable: true,
+				configurable: true,
+				writable: true,
+				value: globals[name]
+			});
+		} else if (desc.writable) {
+			// @ts-expect-error
+			globalThis[name] = globals[name];
+		}
```

Comment:
Should we add a console.warn in case we detect it can't be overridden or just bail out silently? I'm not sure which is better - warnings could result in needless issues opened, no warning could lead to hard-to-debug issues